commit 09a604d85a993ce7589afb375e372e9f23a6a863
Author: Owen W. Taylor <otaylor@fishsoup.net>
Date:   Fri Oct 23 03:40:13 2015 -0400

    Fix reversed check for NULL child
    
    The check for an unknown child path resulting in a NULL child
    added in a614d447f was backwards.
    
    https://bugzilla.gnome.org/show_bug.cgi?id=755951

diff --git a/atspi/atspi-event-listener.c b/atspi/atspi-event-listener.c
index be7d3e9..386393a 100644
--- a/atspi/atspi-event-listener.c
+++ b/atspi/atspi-event-listener.c
@@ -207,7 +207,7 @@ cache_process_children_changed (AtspiEvent *event)
     return;
 
   child = g_value_get_object (&event->any_data);
-  if (child)
+  if (child == NULL)
     return;
 
   if (!strncmp (event->type, "object:children-changed:add", 27))
